src/main/resources/db/migration/V1__init.sql

```diff
@@ -118,20 +117,20 @@ CREATE TABLE IF NOT EXISTS roles (
   id BIGSERIAL NOT NULL,
   name VARCHAR(100) NOT NULL,
   company_id BIGINT NOT NULL,
-  is_deletable BOOLEAN NULL DEFAULT true,
```

Comment:
rename the field `isDeleted` if it doesn't have special meaning.

Reply:
> rename the field `isDeleted` if it doesn't have special meaning.

We already have an active field, do we need `isDeleted` then?

Reply:
I have a little doubt here, so lets keep it simple, if we want to use it for deleted feature then keep `isDeleted` and if you want to use to show active/in-active on screen keep `isActive` but remember record will still appear on screen, in case of in active and for isDeleted it will not appear on screen, so do give a though this table needs `isActive` kind of feature if yes, then `isDeleted` not required and if no. then `isActive` not required.